modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/SampleTimelineFDSView.java

```diff
@@ -25,7 +25,7 @@
  */
 @Component(
 	enabled = false,
-	property = "frontend.data.set.name=" + FDSSampleFDSNames.FDS_SAMPLES,
+	property = "frontend.data.set.name=" + FDSSampleFDSNames.CUSTOMIZED_FDS_SAMPLE,
 	service = FDSView.class
 )
 public class SampleTimelineFDSView extends BaseTimelineFDSView {
```

Comment:
Does this class name should reference the customized sample, like in CustomizedSampleTableFDSView?